src/test/java/com/synopsys/integration/blackduck/codelocation/signaturescanner/ScannerZipInstallerTest.java

```diff
@@ -38,19 +40,17 @@
 import com.synopsys.integration.util.OperatingSystemType;
 
 @ExtendWith(TimingExtension.class)
-public class ScannerZipInstallerTest {
+class ScannerZipInstallerTest {
     @Test
-    public void testActualDownload() throws Exception {
-        IntEnvironmentVariables intEnvironmentVariables = IntEnvironmentVariables.includeSystemEnv();
-
-        String signatureScannerDownloadPath = intEnvironmentVariables.getValue("BLACKDUCK_SIGNATURE_SCANNER_DOWNLOAD_PATH");
-        String blackDuckUrl = intEnvironmentVariables.getValue("BLACKDUCK_URL");
-        String blackDuckUsername = intEnvironmentVariables.getValue("BLACKDUCK_USERNAME");
-        String blackDuckPassword = intEnvironmentVariables.getValue("BLACKDUCK_PASSWORD");
-        assumeTrue(StringUtils.isNotBlank(signatureScannerDownloadPath));
-        assumeTrue(null != blackDuckUrl);
-        assumeTrue(StringUtils.isNotBlank(blackDuckUsername));
-        assumeTrue(StringUtils.isNotBlank(blackDuckPassword));
+    void testActualDownload() throws Exception {
+        String signatureScannerDownloadPath = TestingPropertyKey.TEST_BLACKDUCK_SIGNATURE_SCANNER_DOWNLOAD_PATH.fromEnvironment();
+        String blackDuckUrl = TestingPropertyKey.TEST_BLACK_DUCK_SERVER_URL.fromEnvironment();
+        String blackDuckUsername = TestingPropertyKey.TEST_USERNAME.fromEnvironment();
+        String blackDuckPassword = TestingPropertyKey.TEST_PASSWORD.fromEnvironment();
+        assertTrue(StringUtils.isNotBlank(signatureScannerDownloadPath));
+        assertTrue(StringUtils.isNotBlank(blackDuckUrl));
+        assertTrue(StringUtils.isNotBlank(blackDuckUsername));
+        assertTrue(StringUtils.isNotBlank(blackDuckPassword));
```

Comment:
I'm tempted to remove these assertions and let the validator fail the test. Thoughts?

Reply:
Agreed